lesson-3/Dooley - Lesson 3/basketballs-and-confused/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>both</title>
+</head>
+<body>
+	<h1>troy bolton with a basketball looking confused</h1>
+
+	<img src="../images/troy-basketball-confused.jpg" width="300px" />
+
+	<p>Finished looking at this Troy Bolton image? Choose where to go next:</p>
+
+	<ul>
+		<li>
+			<a href="../index.html">Home</a>
```

Comment:
Nice! This works great since it links directly to the page, however, because you've named your files `index.html` your browser will automatically find them, and you can actually just write `../` here. 

Same with the ones below, they can be written as `../basketballs/` etc.

This will become more apparent once we're working with live websites rather than ones that are local only to our computers.